ramls/invoice.raml

```diff
@@ -55,6 +58,32 @@ resourceTypes:
           schema: invoice
           exampleItem: !include acq-models/mod-invoice-storage/examples/invoice.sample
 
+      /documents:
+        displayName: Document
+        description: Manage documents assoctiated with invoice
+        type:
+          collection:
+            schemaCollection: documentCollection
+            schemaItem: invoiceDocument
+            exampleCollection: !include acq-models/mod-invoice-storage/examples/document_collection.sample
+            exampleItem: !include acq-models/mod-invoice-storage/examples/document.sample
+        get:
+          description: Get list of documents
+          is: [
+            pageable,
+            searchable: {description: "with valid searchable fields: for example metadata.createdDate", example: "metadata.createdDate > '2018-07-19T00:00:00.000+0000'"}
+          ]
+        /{documentId}:
+          description: Get and Delete a specific invoice. Update operation is not allowed
```

Comment:
probably want to fix this description... we're not getting/deleting invoices, but invoice document attachments/links

Reply:
fixed